src/file.ts

```diff
@@ -435,6 +435,7 @@ class File extends ServiceObject {
    * Another example:
    */
   copy(destination: string|Bucket|File, callback: FileCallback);
+  copy(destination: string|Bucket|File, callback: FileCallback);
```

Comment:
I am confused.  This looks like the same declaration twice?

Reply:
My bad. That wasn't suppose to be there.